editor/src/clj/editor/code/view.clj

```diff
@@ -994,12 +994,13 @@
       :text)
     display-string))
 
-(defn- update-document-width
-  [evaluation-context view-node]
-  (when-let [resource-node (g/maybe-node-value view-node :resource-node)]
+(defn- update-document-width [evaluation-context view-node]
+  ;; This is called in response to changing layout-related properties on the
+  ;; view-node. We only want to do something when we're connected to a resource
+  ;; node. If we're not, the lines output will return nil.
+  (when-some [lines (g/node-value view-node :lines evaluation-context)]
```

Comment:
The `resource-node` output will not be valid until the `app-view` has connected the `view-node` that we're constructing, so instead we rely on the fact that the `lines` input will not have anything connected until we have established the connection to the `resource-node`. This allows us to call this function from inside the `setup-view!` function.